raiden/network/transport/matrix/client.py

```diff
@@ -104,21 +104,22 @@ def update_local_aliases(self) -> bool:
         """
         server_name = urlparse(self.client.api.base_url).netloc
         changed = False
-        for event_type in ["m.room.aliases", "m.room.canonical_alias"]:
```

Comment:
why don't we need the canonical aliases anymore?

Reply:
we don't use them anymore